src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -461,6 +465,34 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           StandardOutputImportance="High" />
   </Target>
 
+  <Target Name="_GenerateComposites"
+    DependsOnTargets="_ExpandRuntimePackageRoot">
+
+    <PropertyGroup>
+      <CompositeTargetDir>$(TargetDir)Composite\</CompositeTargetDir>
+
+      <CompositeTargetOs>$(TargetOsName)</CompositeTargetOs>
+      <CompositeTargetOs Condition="'$(TargetOsName)' == 'win'">windows</CompositeTargetOs>
+      <CompositeTargetOs Condition="'$(TargetOsName)' == 'linux-musl'">linux</CompositeTargetOs>
```

Comment:
This target runs after `_BatchCrossGenAssemblies`. Why not reuse `$(Crossgen2TargetOs)`❔